src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -321,7 +323,8 @@ class Microsoft::Terminal::Core::Terminal final :
 
     // TODO: These members are not shared by an alt-buffer. They should be

```

Comment:
does this PR change the state of this todo?

Reply:
consider renaming `_mutableViewport` if you haven't already, so we can detect other places where it was being used